flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js

```diff
@@ -15,6 +15,12 @@ class A320_Neo_CDU_MainDisplay extends FMCMainDisplay {
         this._lines = [];
         this._keypad = new Keypad(this);
         this.scratchpad = null;
+        this._inout = null;
```

Comment:
```suggestion
        this._inOut = null;
```

Is this supposed to be ``this._inOut`` ?

Reply:
Yes it is. Great catch!